src/ui-core/templates/pages/CategoryPage.template.tsx

```diff
@@ -1,15 +1,11 @@
 import React from "react";
-import ProductCard from "../../components/productCard/ProductCard.tsx";
-import Heading from "../../components/typography/Heading.tsx";
-import "../../components/productCard/ProductCard.css";
-import "../../components/typography/Heading.css";
-import useFetchProducts from "../../hooks/useFetchProducts.ts";
-import Loading from "../../components/loading/Loading.tsx";
-import { Product } from "../../models/Product.tsx";
-interface ProductsProps {
-  url: string;
-  descriptionBackgroundColor: string;
-}
+import ProductCard from "../../components/molecules/ProductCard/ProductCard.component.tsx";
+import Heading from "../../components/atoms/Typography/Heading.component.tsx";
+import "../../components/molecules/ProductCard/ProductCard.component.css";
+import "../../components/atoms/Typography/Heading.component.css";
```

Comment:
@N-i-z This should be imported inside the `Heading` component—if it has styles specific to this template, create a separate style sheet or add it inline!